R/lazy-join-query.R

```diff
@@ -171,7 +173,8 @@ sql_build.lazy_multi_join_query <- function(op, con, ..., sql_options = NULL) {
   op$joins$by <- purrr::map2(
     op$joins$by, seq_along(op$joins$by),
     function(by, i) {
-      by$x_as <- table_names_out[op$joins$by_x_table_id[[i]]]
+      # @mgrlich: x_as should be a single value, right?
+      by$x_as <- table_names_out[unique(op$joins$by_x_table_id[[i]])]
```

Comment:
As `x_as` can have length > 1 the unique is wrong.

```suggestion
      by$x_as <- table_names_out[op$joins$by_x_table_id[[i]]]
```